modules/Nets/staNNet.py

```diff
@@ -26,17 +26,19 @@
 import pdb
 
 class staNNet(object):
-    
-    def __init__(self,*args,loss='MSE',activation='ReLU', dim_cv=5, BN=False):
+
+    def __init__(self,*args,loss='MSE',activation='ReLU', dim_cv=5, BN=False, conversion=100):
         
         if len(args) == 3: #data,depth,width
            data,depth,width = args           
-           self.info = {'activation':activation, 'loss':loss}
+           self.info = {'activation':activation, 'loss':loss, 'conversion':conversion}
```

Comment:
'conversion' should be a key already in the meta data of the data file, right?

Reply:
'conversion' and 'amplification' are two different things and the two might get confusing.

'amplification' (which is in the metadata) is the factor used on the output data to rescale the outputs to terms of nA. Since the IV converter in the measurement set up can only measure between +- 3.7V, there is a certain amplification one can apply to map the [-3.7 , 3.7]V range to either nA, tens of nA, hundreds of nA or microAmps. Since we want to have our outputs always in the same units, we multiple our [-3.7 , 3.7]V range with the correct amplification to obtain the output in nA.

'conversion' is a purely artificial term which right now is used to scale the outputs (which are in nA) to a range that corresponds to the order of magnitude of the inputs. This does not necessarily have to be the same factor as 'amplification'.

Maybe to make this all less confusing I could remove the 'conversion' term and during training always divide again by 'amplification'? This way the NN trains on outputs in range [-3.7 , 3.7] and the inputs are in terms of volts. Although this removes some flexibility on scaling, the input and output range will for most cases be roughly similar so the weight updates should be fine.